common/modules/gcp-external-backup/main.tf

```diff
@@ -0,0 +1,62 @@
+terraform {
+  backend "gcs" {}
+}
+
```

Comment:
I think this module is missing the provider config block - https://github.com/gpii-ops/gpii-infra/blob/c2ef3a02cdccfd6a0e2c2bdfe9137bc749f01e6c/common/modules/backup-storage-bucket/main.tf#L10-L12

Reply:
solved below